benchmarks/functional_autograd_benchmark/audio_text_models.py

```diff
@@ -0,0 +1,128 @@
+import torch
+import torch.nn as nn
+
+import torchaudio_models as models
+
+from utils import make_functional, load_weights
+
+def get_wav2letter(device):
+    N = 10
+    input_frames = 700
+    vocab_size = 28
+    model = models.Wav2Letter(num_classes=vocab_size)
+    criterion = torch.nn.NLLLoss()
+    model.to(device)
+    params, names = make_functional(model)
+
+    inputs = torch.rand([N, 1, input_frames], device=device)
+    labels = torch.rand(N, 3, device=device).mul(vocab_size).long()
+
+    def forward(*new_params):
+        load_weights(model, names, new_params)
+        out = model(inputs)
+
+        loss = criterion(out, labels)
+        return loss
+
+    return forward, params
+
+def get_deepspeech(device):
+    supported_rnns = {
+        'lstm': nn.LSTM,
+        'rnn': nn.RNN,
+        'gru': nn.GRU
+    }
```

Comment:
Why provide the `supported_rnns` list if the code below only uses `"lstm"`?

Reply:
Copy pasted from example. I will clean it up!